gpii/node_modules/deviceReporter/test/configs/live.json

```diff
@@ -0,0 +1,70 @@
+{
+    "type": "gpii.tests.deviceReporter.config.live",
+    "options": {
+        "components": {
+          "deviceReporterMock": {
+              "type": "gpii.test.integration.deviceReporterAware",
+              "options": {
+                  "mockDeviceReporters": {
+                      "gpii.packageKit.find": {
+                          "mockFunc": "findNameInList",
+                          "defaults": {
+                              "gradeNames": "fluid.function",
+                              "argumentMap": {
+                                  "name": 0
+                              }
+                          }
+                      }
+                  },
+                  "deviceReporters": {
+                      "gpii.packageKit.find": {
+                          "expectInstalled": [
```

Comment:
I find the "expectInstalled" wildly misleading - since this is rather what we "report as installed"... But looking at the acceptance tests, I can see that the reason for this naming, since in the case of acceptance tests these are indeed what we expect to be installed... I might give you and Bosmon a ping in the channel when you're both there to discuss if we can do something clever.. But the actual changes can be done here: https://github.com/GPII/universal/pull/494 and doesn't need to be adressed in this pull request